package.json

```diff
@@ -39,6 +39,7 @@
     "css-loader": "^0.28.7",
     "eslint": "^4.13.1",
     "eslint-plugin-vue-libs": "^2.0.1",
+    "express-open-in-editor": "^3.1.1",
```

Comment:
why are we using the express middleware instead of `open-in-editor` itself?

Reply:
It's to be used in the webpack-dev-server directly (that uses an express server).

Reply:
See https://github.com/lahmatiy/express-open-in-editor#using-with-webpack-dev-server